src/util/system.h

```diff
@@ -148,6 +148,8 @@ class ArgsManager
     std::string m_network GUARDED_BY(cs_args);
     std::set<std::string> m_network_only_args GUARDED_BY(cs_args);
     std::map<OptionsCategory, std::map<std::string, Arg>> m_available_args GUARDED_BY(cs_args);
+    std::set<std::string> m_config_sections GUARDED_BY(cs_args);
+    std::set<std::string> m_available_sections GUARDED_BY(cs_args);
```

Comment:
This could be a static const variable in system.cpp?

Reply:
Yes. changed to a static const valiable.